bin/getInfoRippled.sh

```diff
@@ -0,0 +1,47 @@
+ #!/bin/bash
+
+export TMP_LOC="/tmp/ripple_info"
+if [ ! -d "${TMP_LOC}" ]; then
+       sudo mkdir "${TMP_LOC}"
```

Comment:
I'm not sure why we need the `sudo` here. Can we do without it or is it necessary?